projects/admin/src/app/record/brief-view/circ-policies-brief-view.component.ts

```diff
@@ -32,7 +33,6 @@ import { ResultItem } from '@rero/ng-core';
 })
 export class CircPoliciesBriefViewComponent implements ResultItem {
 
-  @Input()
   record: any;
 
   @Input()
```

Comment:
The @Input for properties "type" and "detailUrl" can also be removed.

Reply:
Done